frontend/src/components/landing/Plans.tsx

```diff
@@ -8,15 +8,15 @@ import {
   getPlan,
   getPremiumSubscribeLink,
   isPremiumAvailableInCountry,
-  RuntimeDataWithPremiumAvailable,
 } from "../../functions/getPlan";
 import { trackPurchaseStart } from "../../functions/trackPurchase";
 import { getRuntimeConfig } from "../../config";
 import Link from "next/link";
 import { isFlagActive } from "../../functions/waffle";
+import { RuntimeData } from "../../hooks/api/runtimeData";
 
 export type Props = {
-  premiumCountriesData?: RuntimeDataWithPremiumAvailable;
+  runtimeData?: RuntimeData;
```

Comment:
Most of the lines changed are the rename from `premiumCountriesData` to `runtimeData`, and updating the type to just `RuntimeData` (rather than a type that indicates that we've already checked that Premium is available).

The actual fix, however, is in `index.page.ts` and `premium.page.ts`, where this prop is also added for the rendering of `<Plans>` in non-Premium countries.